src/pages/markets/components/sections/_other-markets.tsx

```diff
@@ -291,8 +291,8 @@ const OtherMarkets = ({ except }: OtherMarketsProps) => {
                         {localize('Other markets you might be interested in')}
                     </StyledHeader>
                     <Carousel has_autoplay autoplay_interval={4000} {...settings}>
-                        {filteredMarkets.map((market) =>
-                            market === '' ? <div></div> : <Card market={market} key={market} />,
+                        {filteredMarkets.map((market, idx) =>
+                            market === '' ? <div key={idx}></div> : <Card market={market} key={market} />,
```

Comment:
@eduard-deriv, Just as an experiment, what happens if you put `null` here instead of `<div key={idx}></div>`. This is out of the scope of your `PR`, so your changes are Ok, but I just want to know what happens and I'm too lazy to check it out my self and you already have this cloned I guess 😄

Reply:
@jim-deriv , thanks. Change to `null`, was overwhelmed and didn't notice such a simple solution 👍

Reply:
Thank you @eduard-deriv for the confirmation, such changes sometimes break functionality, that's why I'd hesitate  to use it as well 😄